src/sentry/utils/samples.py

```diff
@@ -100,14 +100,19 @@ def load_data(platform, default=None, timestamp=None, sample_name=None):
     #     event so it's not an empty project.
     #   * When a user clicks Test Configuration from notification plugin settings page,
     #     a fake event is generated to go through the pipeline.
-    sample_name = sample_name or platform
+    sample_name = sample_name or INTEGRATION_ID_TO_PLATFORM_DATA[platform]['name']
 
     data = None
-    for platform in (platform, default):
+    language = None
+
+    if INTEGRATION_ID_TO_PLATFORM_DATA.get(platform)['type'] != 'language':
```

Comment:
doing `.get` here does nothing because while you're avoiding a `KeyError`, you're just going to wind up with a `TypeError` from `['type']`